### ISSUE TITLE
[react/jsx-indent] False positive

### ISSUE THREAD

**Issue by Robloche** [yannickcr/eslint-plugin-react#3238]
My code was OK until version 7.28.0 but fails with multiple react/jsx-indent errors in versions 7.29.0-3.

**Example:**
```js
    render(): React.Node {
      const {forwardedRef, isDisabled, key} = this.props;

      return <WrappedComponent
        forwardedRef={forwardedRef}
        isDisabled={isDisabled}
        key={key}
        svgElement={svgElement} />;
    }
```

**Error:**
`ESLint: Expected indentation of 6 space characters but found 8.(react/jsx-indent)`

Applying autofix gives me this:
```js
    render(): React.Node {
      const {forwardedRef, isDisabled, key} = this.props;

      return <WrappedComponent
        forwardedRef={forwardedRef}
        isDisabled={isDisabled}
        key={key}
      svgElement={svgElement} />;
    }
```

which leads to:
`ESLint: Expected indentation of 8 space characters but found 6.(react/jsx-indent-props)`

**Comment by Robloche** [comment]
```
"react/jsx-indent": ["error", 2],
```

**Comment by ljharb** [comment]
Thanks; i reproduced this with failing tests, but wasn’t able to come up with an easy fix. Help is appreciated.

**Comment by saraadams** [comment]
Have you tried formatting as follows?
```
      return <WrappedComponent
        forwardedRef={forwardedRef}
        isDisabled={isDisabled}
        key={key}
        svgElement={svgElement} />
      ;
```

**Comment by Robloche** [comment]
This gives me a new error: `semi-style` (semicolon is expected to be at the end of the previous line)
